app/api/routes.py

```diff
@@ -129,14 +140,17 @@ def handling_incidents(
             f"{target_component} moved from {src_incident.text} to "
             f"{dst_incident.text}"
         )
-        update_incident_status(
-            src_incident,
-            f"{comp_with_attrs} moved to {dst_incident.text}"
-        )
-        update_incident_status(
-            dst_incident,
-            f"{comp_with_attrs} moved from {src_incident.text}"
-        )
+
+        url_d = url_for('web.incident', incident_id=dst_incident.id)
+        url_s = url_for('web.incident', incident_id=src_incident.id)
+        link_s = f"<a href='{url_d}'>{dst_incident.text}</a>"
+        link_d = f"<a href='{url_s}'>{src_incident.text}</a>"
+        update_s = f"{comp_with_attrs} moved to {link_s}"
+        update_d = f"{comp_with_attrs} moved from {link_d}"
```

Comment:
during testing I don't see the details only the strings moved to and moved from
![image](https://github.com/stackmon/status-dashboard/assets/32979462/e49c5250-a289-4b10-ae73-55740a2e9bd2)